ethereum/spec/src/main/java/tech/pegasys/teku/spec/logic/common/helpers/BeaconStateAccessors.java

```diff
@@ -143,6 +143,14 @@ public UInt64 getTotalActiveBalance(BeaconState state) {
             epoch -> getTotalBalance(state, getActiveValidatorIndices(state, epoch)));
   }
 
+  public UInt64 getProposerBoostAmount(final BeaconState state) {
+    final int numValidators = getActiveValidatorIndices(state, getCurrentEpoch(state)).size();
+    final UInt64 avgBalance = getTotalActiveBalance(state).dividedBy(numValidators);
+    final long committeeSize = numValidators / config.getSlotsPerEpoch();
+    final UInt64 committeeWeight = avgBalance.times(committeeSize);
+    return committeeWeight.times(config.getProposerScoreBoost()).dividedBy(100);
```

Comment:
can boost ever be zero? should we just check if it's zero and return 0? not a divisor issue at least i guess

Reply:
It can be zero but it's 70 in every configuration currently.  We'd only set it to 0 to temporarily disable its effects so I don't think it's worth optimising to avoid the calculating in that case.